packages/core/src/dialects/abstract/query-generator.d.ts

```diff
@@ -93,6 +94,11 @@ export interface RemoveColumnQueryOptions {
   ifExists?: boolean;
 }
 
+export interface ShowAllTablesOptions extends QueryRawOptions {
```

Comment:
Shouldn't this belong in `query-interface.d.ts` since it's not used for `showTablesQuery`?

Reply:
Yes, moved `ShowAllTablesOptions` into `query-interface.d.ts`.